diff --git a/cica.py b/cica.py
index 9f7786c..39a64a0 100644
--- a/cica.py
+++ b/cica.py
@@ -13,8 +13,8 @@ from datetime import datetime
 ASCENT = 820
 DESCENT = 204
 SOURCE = os.getenv('CICA_SOURCE_FONTS_PATH', './sourceFonts')
-LICENSE = open('./LICENSE.txt').read()
-COPYRIGHT = open('./COPYRIGHT.txt').read()
+LICENSE = open('./LICENSE.txt', encoding='utf-8').read()
+COPYRIGHT = open('./COPYRIGHT.txt', encoding='utf-8').read()
 VERSION = '5.0.2'
 FAMILY = 'Cica'
 
